packages/ckeditor5-ui/src/autocomplete/autocompleteview.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/autocomplete/autocompleteview
+*/
+
+import { getOptimalPosition, type PositioningFunction, type Locale, global } from '@ckeditor/ckeditor5-utils';
+import SearchView, { type SearchViewConfig } from '../search/searchview';
+import type SearchResultsView from '../search/searchresultsview';
+
+import '../../theme/components/autocomplete/autocomplete.css';
+
+/**
+ * The autocomplete component's view class. It extends the {@link module:ui/search/searchview~SearchView} class
+ * with a floating {@link #resultsView} that shows up when the user starts typing and hides when they blur
+ * the component.
+ */
+export default class AutocompleteView extends SearchView {
```

Comment:
Please note that this view implementation is not complete yet (very basic) as we don't know all the use-cases.